src/textures/bitmap.cpp

```diff
@@ -224,6 +224,7 @@ class BitmapTexture final : public Texture<Float, Spectrum> {
                         exceed_unit_range = true;
                     value = srgb_model_fetch(value);
                     mean += (double) srgb_model_mean(value);
+                    max = max < dr::max(value) ? dr::max(value) : max;
```

Comment:
Though that makes sense, it would mean that max() in the context of bitmap and checkerboard textures mean something different than that of the volume texture where the max represents a per-channel maximum.

Reply:
I've updated the implementation to use the luminance rather than the per-component maximum in line with your comment.